wasm-runtime/polkadot/src/runtime/staking.rs

```diff
@@ -31,7 +32,7 @@ pub type Bondage = u64;
 struct IntentionStorageVec {}
 impl StorageVec for IntentionStorageVec {
 	type Item = AccountID;
-	const PREFIX: &'static[u8] = b"ses:wil:";
+	const PREFIX: &'static[u8] = b"sta:wil:";
```

Comment:
`ses`? `wil`? `sta` = `staking`? Another +1 for contants.

Reply:
i'm planning on abstracting all this in the not-too-distant future to readable static values, but would prefer to leave it to another PR